etc/config-template.toml

```diff
@@ -63,6 +42,25 @@
 ## to release memory.
 # memory-usage-limit = "0B"
 
+[log]
+## Log levels: trace, debug, info, warning, error, critical.
+## Note that `debug` and `trace` are only available in development builds.
+# level = "info"
+## log format, one of json, text. Default to text.
+# format = "text"
+## Enable automatic timestamps in log output, if not set, it will be defaulted to true.
+# enable-timestamp = true
+
+[log.file]
+## Usually it is set through command line.
+# filename = ""
+## max log file size in MB (upper limit to 4096MB)
+# max-size = "300MB"
+## max log file keep days
+# max-days = "0d"
```

Comment:
"0d" means no clean up according to log's timestamp?

Reply:
Yes